makefile

```diff
@@ -21,7 +21,7 @@ ifdef LD_PATH
 	LDFLAGS += $(addprefix -L, $(LD_PATH))
 endif
 
-LDFLAGS += -lphobos2 $(LIBD_LIB) $(LLVM_LIB)
+LDFLAGS += $(LIBD_LIB) -lphobos2 $(LLVM_LIB)
 
 ifeq ($(PLATFORM),Linux)
 	LDFLAGS += -lstdc++ -export-dynamic -ldl -lffi -lpthread -lm -lncurses
```

Comment:
Could you remove the -l here that are now handled by system-libs please ? You can also remove ncurse from the OSX path.